policyengine_us/variables/household/income/person/is_eligible_for_fsla_overtime_protection.py

```diff
@@ -0,0 +1,56 @@
+from policyengine_us.model_api import *
+
+
+class is_eligible_for_fsla_overtime_protection(Variable):
```

Comment:
Do statutes or regs apply the term `protection`? If not can we avoid it or use a more neutral term?

Reply:
or just `is_eligible_for_fsla_overtime`?